discretisedfield/tests/test_field.py

```diff
@@ -495,33 +496,32 @@ def test_norm_zero_field():
     assert np.all(f.norm.array == 0)
 
 
-# TODO Sam
-def test_orientation():
+@pytest.mark.parametrize("nvdim", [1, 2, 3, 4])
+def test_orientation(valid_mesh, nvdim):
+    # No zero-norm cells
+    inital_value = np.zeros(nvdim)
+    inital_value[-1] = 2
+    f = df.Field(valid_mesh, nvdim=nvdim, value=inital_value)
+    assert isinstance(f.orientation, df.Field)
+    assert np.allclose(f.orientation.mean(), inital_value / 2)
```

Comment:
What happens for a 1d field e.g. with values `[-3, -2, 1, 3]`? Do we get `[-1, -1, 1, 1]` for `field.norm`?

Reply:
If we have a field with `nvdim=1` with values `[-3, -2, 1, 3]`, we would get `[3, 2, 1, 3]` for `field.norm` and `[-1, -1, 1, 1]` for `field.orientation`.